recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+source:
+  url: https://github.com/menpo/condaci/raw/master/vs2008_patch.zip
```

Comment:
Should add there are other cases where we are making the same decision (e.g. the `toolchain`, `conda-forge-build-setup`, etc.)